examples/cisco/demo_bgp.pp

```diff
@@ -73,7 +73,11 @@
     enforce_first_as                       => false,
     event_history_cli                      => 'size_small',
     event_history_detail                   => 'size_medium',
+    # event_history_errors is not supported on all platforms
+    # event_history_errors                   => 'size_small',
```

Comment:
I made the changes to get 2 conditions using facts and platform_get(), this is working! But I don't know if there is a better way, I tried other combinations but I am getting syntax failures. If there are more than 2 conditions, we can keep creating loops inside but not sure if that looks good but at least there seems to be a way. If you have a better suggestion, let me know, I can try.